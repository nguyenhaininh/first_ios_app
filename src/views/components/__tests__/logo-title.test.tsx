/*---------------------------------------------------------------------------------------------
 *  Copyright (c) Lunascape Corporation. All rights reserved.
 *--------------------------------------------------------------------------------------------*/
import * as React from 'react';
import LogoTitle from '../logo-title';

import * as renderer from 'react-test-renderer';

it('renders correctly', () => {
const tree = renderer.create(<LogoTitle/>).toJSON;
  expect(tree).toMatchSnapshot();
});
