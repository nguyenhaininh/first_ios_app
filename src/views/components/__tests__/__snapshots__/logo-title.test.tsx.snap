exports[`renders correctly 1`] = `
<Image
  source={
    Object {
      "process": [Function],
    }
  }
/>
`;
